app/models/vocab/fedora_state.rb

```diff
@@ -0,0 +1,6 @@
+require 'rdf'
+module Vocab
+  class FedoraState < RDF::StrictVocabulary('http://fedora.info/definitions/1/0/access/')
```

Comment:
Minor points, but:

* Should we align the name of this class with the name of the ontology? It's the Fedora Resource Status ontology.
* I believe the URI here may be wrong. you're missing `ObjState#` on the end.